### PR TITLE
Add Identity Two-factor features and settings.

### PR DESCRIPTION
**Pull request by maliming**
Use `DisplayText.ResourceName` to localize options in the `FeatureManagementModal` page

### CODE REVIEW

**Comment by hikalkan** [comment]
Is this still draft?

**Comment by maliming** [comment]
Yes, I will make it done asap.

File: modules/identity/src/Volo.Abp.Identity.Domain.Shared/Volo/Abp/Identity/Features/IdentityFeatureCheckerExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using Volo.Abp.Features;
+
+namespace Volo.Abp.Identity.Features
+{
+    public static class IdentityFeatureCheckerExtensions
```
Comment (hikalkan) [inline modules/identity/src/Volo.Abp.Identity.Domain.Shared/Volo/Abp/Identity/Features/IdentityFeatureCheckerExtensions.cs:8]: An internal extension method would be better to not add a global extension to a framework interface that is only used by the identity module. You can use `InternalsVisibleTo` attribute to share this internal to other identity assemblies.

File: modules/identity/src/Volo.Abp.Identity.Domain.Shared/Volo/Abp/Identity/Localization/tr.json [hunk 2]
```diff
@@ -76,6 +76,12 @@
     "Permission:ChangePermissions": "İzinleri değiştirme",
     "Permission:UserManagement": "Kullanıcı yönetimi",
     "Permission:UserLookup": "Kullanıcı sorgulama",
+    "Feature:IdentityGroup": "Kimlik",
+    "Feature:TwoFactor": "İki faktörlü kimlik doğrulama",
+    "Feature:TwoFactorDescription": "İki faktörlü kimlik doğrulama",
+    "Feature:TwoFactor.Optional": "İsteğe bağlı",
+    "Feature:TwoFactor.Disabled": "Devre dışı",
+    "Feature:TwoFactor.Forced": "Zorla etkinleştirildi",
```
Comment (hikalkan) [inline modules/identity/src/Volo.Abp.Identity.Domain.Shared/Volo/Abp/Identity/Localization/tr.json:84]: `Zorunlu` would be enough :)

File: modules/identity/src/Volo.Abp.Identity.Domain.Shared/Volo/Abp/Identity/Localization/tr.json [hunk 3]
```diff
@@ -103,6 +109,10 @@
     "Description:Abp.Identity.SignIn.EnablePhoneNumberConfirmation": "Oturum açmak için telefon numarası gerekli",
     "Description:Abp.Identity.SignIn.RequireConfirmedPhoneNumber": "Oturum açmak için onaylanmış bir telefon numarasının gerekli olup olmadığı.",
     "Description:Abp.Identity.User.IsUserNameUpdateEnabled": "Kullanıcı adının, kullanıcının kendisi tarafından güncellenebilirliği.",
-    "Description:Abp.Identity.User.IsEmailUpdateEnabled": "E-posta alanının, kullanıcının kendisi tarafından güncellenebilirliği"
+    "Description:Abp.Identity.User.IsEmailUpdateEnabled": "E-posta alanının, kullanıcının kendisi tarafından güncellenebilirliği",
+    "DisplayName:Abp.Identity.TwoFactorBehaviour": "İki faktörlü kimlik doğrulama davranışı",
+    "Description:Abp.Identity.TwoFactorBehaviour": "İki faktörlü kimlik doğrulama davranışı",
+    "DisplayName:Abp.Identity.UsersCanChange": "Kullanıcıların faktör kimlik doğrulamasını değiştirmesine izin verin.",
```
Comment (hikalkan) [inline modules/identity/src/Volo.Abp.Identity.Domain.Shared/Volo/Abp/Identity/Localization/tr.json:115]: `Kullanıcıların iki faktörlü kimlik doğrulama davranışını değiştirmelerine izin verin`

File: modules/identity/src/Volo.Abp.Identity.Domain.Shared/Volo/Abp/Identity/Localization/tr.json [hunk 4]
```diff
@@ -103,6 +109,10 @@
...
+    "Description:Abp.Identity.UsersCanChange": "Kullanıcıların faktör kimlik doğrulamasını değiştirmesine izin verin."
```
Comment (hikalkan) [inline modules/identity/src/Volo.Abp.Identity.Domain.Shared/Volo/Abp/Identity/Localization/tr.json:116]: `Kullanıcıların iki faktörlü kimlik doğrulama davranışını değiştirmelerine izin verin`

File: modules/identity/src/Volo.Abp.Identity.Domain/Volo/Abp/Identity/IdentitySettingProviderExtensions.cs [hunk 5]
```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using Volo.Abp.Identity.Features;
+using Volo.Abp.Identity.Settings;
+using Volo.Abp.Settings;
+
+namespace Volo.Abp.Identity
+{
+    public static class IdentitySettingProviderExtensions
```
Comment (hikalkan) [inline modules/identity/src/Volo.Abp.Identity.Domain/Volo/Abp/Identity/IdentitySettingProviderExtensions.cs:10]: Same thought with my IFeatureChecker extension comment ;)

**Review by hikalkan — COMMENTED** [review]
I am merging this, you can make the changes separately ;)

**Pull request merged**